demo/apisix/conf/apisix.yaml

```diff
@@ -6,6 +6,82 @@ routes:
       nodes:
       type: roundrobin
     plugin_config_id: 1
+  -
+    id: jmap
+    uri: /jmap
+    service_id: jmap_service
+    methods:
+      - POST
+  -
+    id: jmap_session
+    uri: /jmap/session
+    service_id: jmap_service
+    methods:
+      - GET
+  -
+    id: download
+    uri: /download/*
+    service_id: jmap_service
+    methods:
+      - GET
+      - OPTIONS
+  -
+    id: upload
+    uri: /upload
+    service_id: jmap_service
+    methods:
+      - POST
+      - OPTIONS
```

Comment:
Q: Why do we need this route? Is it not that the upload route always comes with `/upload/accoundId`?

Reply:
- I reference from `krakend.json` configuration file.
- I quickly check the DownloadRoute and see `/upload` endpoint with Options method (NOT see OPTION method in DownloadContract, not sure here)